Items/04_00_WeaponAttachments.txt

```diff
@@ -694,6 +694,17 @@ using Vorpal rounds, your gun deals 80 + 20 (for Critical) + 8 (Vorpal damage) =
 If your roll an Extra Critical Hit, you would deal 80 +  30(for Extra Critical) 
 + 24 (Vorpal Damage) = 134 damage.
 
+== Naphoxic Gel ($1.5 x mag cost) == 
+    
+    Naphitic gel with small oxygen capsules dispersed inside. Standard flame thrower 
+gel requires atmospheric oxygen to burn, and as such cannot be used in a vacuum. 
+Naphoxic gel can be burnt in any environment, including space and underwater. 
+While in atmospheres with oxygen, it burns hotter than normal gel. The added oxygen
+in the tank reduces its overall fuel capacity.

```

Comment:
"reduces fuel capacity" This is brilliant! It's so simple, so obvious, but I'd have never thought of it! And It plays right into one of the few drawbacks of flamethrowers as they are already! Genius!